tools/configen/configen/config.py

```diff
@@ -10,6 +10,7 @@
 class ModuleConf:
     name: str = MISSING
     classes: List[str] = MISSING
+    _convert_: Optional[str] = None
```

Comment:
Let's add a default_flags dict:
Let's add a defaults_flag dict instead. this also support recursive.
```suggestion
class ModuleConf:
    name: str = MISSING
    classes: List[str] = MISSING
    default_flags: Dict[str, Any] = field(default_factory=dict)
```